pipeline/inputs/windows-event-log-winevtlog.md

```diff
@@ -39,6 +40,12 @@ Here is a minimum configuration example.
 
 Note that some Windows Event Log channels \(like `Security`\) requires an admin privilege for reading. In this case, you need to run fluent-bit as an administrator.
 
+#### Query Languages for Event_Query Parameter
+
+`Event_Query` parameter can handle XML query for filtering Windows EventLog when collecting them.
```

Comment:
```suggestion
`Event_Query` parameter can be used to generate XML query for filtering Windows EventLog during collection.
```

Reply:
What is an XML query though? That's probably the main thing - XML is a generic marshalling tech so is this something specific?

Reply:
I have no idea to describe simply about XML query - which can be composable and declarative format of query language for filtering Windows EventLog.

XML Query language is described here: https://learn.microsoft.com/en-us/windows/win32/wes/consuming-events
But, this is the introductory and not fundamental information of the query language.

Reply:
Ah ok, so XML query is not some special MS thing - it just means query via XML using XPath.
```suggestion
The `Event_Query` parameter can be used to specify the XML query for filtering Windows EventLog during collection.
```